flake.nix

```diff
@@ -32,7 +32,17 @@
                   inherit (prev.darwin.apple_sdk.frameworks) Security;
                 };
 
-                clightning = prev.clightning.overrideAttrs (oldAttrs: {
+                # syncing channels doesn't work right on newer versions, exactly like described here
+                # https://bitcoin.stackexchange.com/questions/84765/how-can-channel-policy-be-missing
+                # note that config-time `--enable-developer` turns into run-time `--developer` at some
+                # point
+                clightning = prev.clightning.overrideAttrs (oldAttrs: rec {
+                  version = "23.05.2";
+                  src = prev.fetchurl {
+                    url = "https://github.com/ElementsProject/lightning/releases/download/v${version}/clightning-v${version}.zip";
+                    sha256 = "sha256-Tj5ybVaxpk5wmOw85LkeU4pgM9NYl6SnmDG2gyXrTHw=";
+                  };
+                  makeFlags = [ "VERSION=v${version}" ];
```

Comment:
Do you know if there's an issue in the lightning repo yet? I cannot find anything but also lack a bit of context. Is this about slower channel sync, i.e. that it takes longer for all the channel policies to be announced?

Reply:
I asked ton Discord #support channel and I don't think I got any answer. Will need to follow-up.